src/Context/Propagation/NullPropagator.php

```diff
@@ -5,6 +5,7 @@
 namespace OpenTelemetry\Context\Propagation;
 
 use OpenTelemetry\Context\Context;
+use OpenTelemetry\Context\ContextInterface;
 
 class NullPropagator implements TextMapPropagatorInterface
```

Comment:
Should be removed, duplicates `NoopTextMapPropagator`.

Reply:
agree